release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -76,9 +76,86 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 ### Runtime updates
 
+#### New features
+
+##### Strand dump
+
+Allows to get the status of strands and strand groups during the execution of a Ballerina program.
+
+This can be used to troubleshoot runtime errors. Ballerina runtime will emit the strand dump to the standard output stream in the text format, if it receives a `SIGTRAP` signal. (`SIGTRAP` is not available on Windows.)
+
+e.g., If the PID of the running Ballerina program is `$PID`, you can get the strand dump by executing either `kill -SIGTRAP $PID` or `kill -5 $PID` command.
+
+##### StopHandler
+
+Allows to register a function that will be called during graceful shutdown.
+
+A call to `onGracefulStop` will result in one call to the handler function that was passed as an argument; the handler functions will be called after calling `gracefulStop` on all registered listeners, in reverse order of the corresponding calls to `onGracefulStop`.
+
+e.g., A function `foo` can be called during the graceful shutdown by registering it as follows,
+
+`runtime:onGracefulStop(foo);`
+
+#### Improvements
+
+#### New Runtime Java APIs
+
+##### Type reference type support at runtime
```

Comment:
```suggestion
##### Type reference type support at runtime
```
This section seems to be empty. Please add the content or remove it if it is not required. Also, the repetition of the word `type` makes it a bit confusing. Shall we try rewording the title?

Reply:
Under the `Type reference type support at runtime` section, we have two sub-sections. 
- `Modified existing runtime APIs`
- `New runtime Java API`
So I think we can keep the title and change it to `Type reference support at runtime`

Reply:
The section `Type reference type support at runtime` have subsections.